src/task-7.js

```diff
@@ -1,6 +1,34 @@
+// import { max } from "lodash";
```

Comment:
The result of `getPolynomial` is a single string, so from this I can tell that `reduce` will work here. Please refactor within specific practice rules in mind. And not cheating like `map` which is actually `forEach`.

> Вместо этого используйте методы массивов map, filter, reduce, sort, every, some